src/site/content/en/blog/text-fragments/index.md

```diff
@@ -379,6 +379,38 @@ Feature detection is mainly intended for cases where links are dynamically gener
 (for example by search engines) to avoid serving text fragments links
 to browsers that do not support them.
 
+### Styling text fragments
+
+By default, browsers style text fragments the same way they style
+[`mark`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/mark)
+(typically black on yellow, the CSS
+[system colors](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#system_colors)
+for `mark`).
```

Comment:
```suggestion
for `mark`.
```

Reply:
This leaves the opening’(‘. Do you want to remove both?